### ISSUE TITLE
App tries to use google or amazon storage buckets even though local storage is selected in Admin dashboard.

### ISSUE THREAD

**Issue by CosmicCoder96** [fossasia/open-event-server#5962]
Could be a docker issue if it works locally, the point is, irrespective of this error being thrown server still returns a garbage corrupt PDF, so it looks like everything is fine, on the frontend, but the tickets don't actually open.

```
FileNotFoundError: [Errno 2] No such file or directory: '/data/app/static/generated/tickets/attendees/tickets/pdf/5c7e4019-b90a-4100-ab01-3746bc25fdcc/U3V2Wmh2bD/5c7e4019-b90a-4100-ab01-3746bc25fdcc.pdf'
```

**Comment by shreyanshdwivedi** [comment]
@CosmicCoder96 this PR was merged 2 days back which fixes the generation of tickets for placed orders too.
https://github.com/fossasia/open-event-server/pull/5957
Idk if this is merged to production too. It might be the reason to this error as we were getting it before this PR.

**Comment by CosmicCoder96** [comment]
@shreyanshdwivedi Everything so far is merged  into production.

**Comment by CosmicCoder96** [comment]
Had to comment out storage helpers for gs bucket and others for it to work, seems like a bug with settings. Will open an issue when I figure out what exactly is breaking.

**Comment by uds5501** [comment]
@CosmicCoder96 @shreyanshdwivedi This could be for the old tickets, actually my PR ensured that the new tickets are being generated, but they didn't handle the case of old non-generated tickets and invoices

**Comment by CosmicCoder96** [comment]
Was fixed in V1.1.0
